nx-X11/lib/X11/XlibInt.c

```diff
@@ -3696,9 +4113,13 @@ _XData32(
  *       and so, you may be better off using gethostname (if it exists).
  */
 
-#if (defined(_POSIX_SOURCE) && !defined(AIXV3) && !defined(__QNX__)) || defined(hpux) || defined(USG) || defined(SVR4)
```

Comment:
It had been removed upstream. I have not bothered removing it all over the place as I was concentrating on the libX11 backport stuff. Your suggestion is valid, will do so.